jest.config.ts

```diff
@@ -8,7 +8,8 @@ const config: Config = {
 	roots: ['<rootDir>/src'],
 	testRegex: '(\\.|/)(test|spec)\\.tsx?$',
 	setupFiles: ['dotenv-expand/config'],
-	clearMocks: true,
+	restoreMocks: true,
```

Comment:
`restoreMocks` is a stricter version of `clearMocks` and allows to catch implicit dependencies among tests.

According to the docs, [clearMocks](https://archive.jestjs.io/docs/en/next/configuration#clearmocks-boolean), in turn:

> Automatically clear mock calls and instances before every test. ... This does **not** remove any mock implementation that may have been provided.

And stricter [restoreMocks](https://archive.jestjs.io/docs/en/next/configuration#restoremocks-boolean):

> Automatically restore mock state before every test. ... This will lead to any mocks having their fake implementations **removed and restores** their initial implementation.

**CC:** @jvrankul23